### PR TITLE
Bug 1138481 - Add .pep8.rc with preferred settings to the repository

### PR DESCRIPTION
**Pull request by IamAdiSri**
Bug 1138481

Created .pep8.rc file
Edited .travis.yml file

### CODE REVIEW

**Comment by IamAdiSri** [comment]
@whimboo r?

**Comment by whimboo** [comment]
Hey @IamAdiSri! Thank you for uploading the patch as PR. Sorry that I haven't had the time to check it earlier today. But I think I can wait with a deeper review given that the Travis CI test failed. This is something you will have to fix first. But let me have a look at it now.

File: .pep8.rc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[pep8]
+max-line-length = 99
+show-source = enable
+statistics = enable
+count = enable
```
Comment (whimboo) [inline .pep8.rc]: As I have mentioned already on the bug in Bugzilla we don't want those three boolean keys included. Those are not necessary and seem to also cause the failure with Travis.

File: .pep8.rc [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+exclude = firefox_pupprteer/docs
```
Comment (whimboo) [inline .pep8.rc]: There is still the typing error included. Please fix that and also add an empty final line to that file.

**Comment by IamAdiSri** [comment]
@whimboo r?
Sorry about the typo. It was callous of me to not have checked the file before committing. The build seems to be okay now.

**Comment by whimboo** [comment]
That looks great now. Thank you for the updates @IamAdiSri. I will get the commits squashed and merged immediately.

**Comment by whimboo** [comment]
Merged as https://github.com/mozilla/firefox-ui-tests/commit/9e20028c7a3df09b162a14667416eed177e35e52.

**Pull request closed without merging**